### PR TITLE
Add sivo variant

### PR DESCRIPTION
**Pull request by nazywam**

### CODE REVIEW

**Review by msm-code — APPROVED** [review]

File: decryptor.py [hunk 1]
```diff
@@ -21,13 +21,15 @@
...
+    "Sivo-README.txt",
 ]
-EXTENSIONS = [".mapo", ".deto", ".mbit", ".dante", ".edab", ".edab1", ".gomer"]
+EXTENSIONS = [".mapo", ".deto", ".mbit", ".dante", ".edab", ".edab1", ".gomer", ".sivo"]
 VERSIONS = {
     "Key verify": "=+ Key verify =+\n((?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?)",
     "L2 Protection": "~ L2 Protection ~\n((?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?)",
     "EDAB": "~ EDAB ~\n((?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?)",
     "GOMER": "~ GOMER ~\n((?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?)",
+    "Key verify": "~ Key verify ~\n((?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?)",
```
Comment (msm-code) [inline decryptor.py]: wait, I ACCed too fast - dupliucated key?

**Review by msm-code — CHANGES_REQUESTED** [review]

File: decryptor.py (hunk 1 above)
Comment (msm-code) [reply decryptor.py]:
```
>>> VERSIONS = {
...     "Key verify": "=+ Key verify =+\n((?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?)",
...     "Key verify": "~ Key verify ~\n((?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?)",
... }
>>> VERSIONS
{'Key verify': '~ Key verify ~\n((?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?)'}
```
thanks python :weltschmerz:

**Review by msm-code — CHANGES_REQUESTED** [review]

File: decryptor.py (hunk 1 above)
Comment (nazywam) [reply decryptor.py]: Oh yeah, good catch :thinking:

**Review by msm-code — APPROVED** [review]
looks good to me

**Pull request merged**